static/app/components/onboardingWizard/onboardingCard.tsx

```diff
@@ -174,6 +174,7 @@ const StyledIconClose = styled(IconClose)`
   position: absolute;
   right: ${space(1.5)};
   top: ${space(1.5)};
+  cursor: pointer;
```

Comment:
Can we switch this to a button? We shouldn't be adding `onClick` to svg icons

Reply:
probably want `<CloseButton borderless size="zero" icon={<IconClose color="gray300" />} />` and then the `CloseButton` has some of these styles

Reply:
Want me to change the other one here as well? https://github.com/getsentry/sentry/blame/7c90d3ea2fa41326738f1b13821b010cf040bafa/static/app/components/onboardingWizard/task.tsx#L127

Reply:
Yeah, let's do it in a sep PR

Reply:
Will let someone else review the rest of this PR